daft/table/table_io.py

```diff
@@ -237,24 +237,10 @@ def read_sql(
         MicroPartition: MicroPartition from SQL query
     """
 
-    if sql_options.predicate_expression is not None:
```

Comment:
Are these `sql_options` still used anywhere then? If not, should we also clean that up?

Reply:
Will remove it